src/app/modules/parts/components/edit-part/edit-part.component.html

```diff
@@ -0,0 +1,49 @@
+<div class="container">
+    <form [formGroup]="editForm" (ngSubmit)="submitForm()">
+      <div class="is-grouped">
+        <label class="label">Название</label>
+        <div class="field has-addons">
+          <div class="control is-expanded">
+            <input class="input" type="text" formControlName="title" placeholder="Название">
+          </div>
+          <div class="control" *ngIf="isEditAvailable()">
+            <a class="button is-info" (click)="onEditClick()">
+              Изменить
+            </a>
+          </div>
+        </div>
+        <p class="help is-warning" *ngIf="editForm.get('title').touched && editForm.hasError('required', ['title'])">Название не должно быть пустым</p>
```

Comment:
not use such complex condition in template.
Can you create method `isFormHasError` or similar